configs/machines/mistral.yaml

```diff
@@ -6,14 +6,14 @@ additional_flags: "--mem=0"
 account: None
 
 choose_general.use_hyperthreading:
-        "0":
-                hyperthreading_flag: "--ntasks-per-core=1"
-        0:
-                hyperthreading_flag: "--ntasks-per-core=1"
-        False:
-                hyperthreading_flag: "--ntasks-per-core=1"
-        "*":
+        "1":
                 hyperthreading_flag: ""
+        1:
+                hyperthreading_flag: ""
+        True:
+                hyperthreading_flag: ""
+        "*":
+                hyperthreading_flag: "--ntasks-per-core=1"
```

Comment:
Have you tried this lines on `mistral`? My guess is that they will break as `1` and `True` are considered as the same once evaluated by the parser and tagged as duplicates. `"1"` and `True` should cause no issue.

Reply:
Just tried it now again, it did not seem to complain....but it also didn't turn the flag on, I'll send you another commit in a bit.

Reply:
The flag does not need to be on for it to complain. So I would leave it as it is (including the `1`).

Reply:
OK, I fixed it now...

Also, the "tidy done" that I had was wrong earlier, the "maybe resubmit" kills off any steps that might happen afterward.